src/track/beats.cpp

```diff
@@ -354,8 +382,13 @@ QByteArray BeatsInternal::toProtobuf() const {
     // items in adjacent memory locations.
     track::io::Beats beatsProto;
 
-    for (int i = 0; i < m_beats.size(); ++i) {
-        beatsProto.add_beat()->CopyFrom(m_beats[i]);
+    for (const auto& m_beat : m_beats) {
+        beatsProto.add_beat()->CopyFrom(m_beat);
+    }
```

Comment:
```suggestion
    for (const auto& beat : m_beats) {
        beatsProto.add_beat()->CopyFrom(beat);
    }
```

`m_` prefix is for member variables, not local variables.